app/models/review_immunity_detail.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class ReviewImmunityDetail < ApplicationRecord
-  DECISIONS = %w[yes no].freeze
+  DECISIONS = %w[Yes No].freeze
```

Comment:
Very minor, but is it possible for these to be symbols? It might not be worth the effort to convert them, but it seems like they're internal identifiers rather than user-visible text, which would make symbols the technically-correct option I think.

Having to convert the case in order to check seems slightly suboptimal.